resources/assets/icons/stratis.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 518 483" xmlns="http://www.w3.org/2000/svg"><path d="M10.055 317c79.167 50.333 238.3 151.5 241.5 153.5 4 2.5 9.5 2 13 0s230-148 236-152 7-7 7-10v-57c0-5.5-4.5-6-7-4s-233 148.5-237.5 151.5-9.5 3-13.5 0-228.5-146.5-233.5-150.5-6-7.5-6-12.5v-51.5c0-5.418-.522-9.792 1.64-13.5m0 0c.83-1.425 2.056-2.751 3.86-4 6.5-4.5 223.5-146.5 232.5-152s12.5-6 21 0c75.5 49.5 227.3 149 230.5 151 4 2.5 5 7-4 12.5s-228 145-231.5 147-7 3-12 0c-4-2.4-161.907-104-240.36-154.5z" stroke="currentColor" stroke-width="20" stroke-linecap="round" stroke-linejoin="round" fill="none"/></svg>
```

Comment:
move it to the networks folder, it's not a regular icon

Reply:
I did not know that, it's moved now!